CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+Version 0.3.19
+-------------
+* Fix swift cleanup of statements after return 
+* Introduce spark3 upgrade plugin 
```

Comment:
```suggestion
```

Reply:
we are not releasing this as part of polyglot-piranha wheel. That is a stand alone.